### PR TITLE
Traffic law in the United States

### PR DESCRIPTION
**Pull request by mburke5678**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.

**Describe the contribution to the taxonomy**

- ...
- ...
- ...

**Input given at the prompt**

```
   ...
```

**Response from the original model**

```
  ...
```

**Response from the fine-tuned model**

```
  ...
```

**Contribution checklist**

- [ ] The contribution was tested with `ilab generate`
- [ ] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [ ] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ ] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ ] Content does not include PII or otherwise sensitive or confidential information
- [ ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

File: knowledge/laws/traffic_laws_us/qna.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+created_by:
```
Comment (jjasghar) [inline knowledge/laws/traffic_laws_us/qna.yaml:1]: Please add your GitHub username here.

File: knowledge/laws/traffic_laws_us/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: traffic_laws
+seed_examples:
+- answer: | Pedestrian crossings are strips of the road where pedestrians are expected to cross
```
Comment (jjasghar) [inline knowledge/laws/traffic_laws_us/qna.yaml:4]: Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!

**Review by jjasghar — CHANGES_REQUESTED** [review]
Please fix these issues then we can run the linter.

**Comment by mburke5678** [comment]
@jjasghar I'm not sure why I cannot push my changes. Is this something you can help with?

```
 $ git push origin mburke5678-patch-1 
Gtk-Message: 09:05:31.257: Failed to load module "canberra-gtk-module"
Gtk-Message: 09:05:35.234: Failed to load module "canberra-gtk-module"
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/mburke5678/taxonomy.git/'
```

**Comment by jjasghar** [comment]
```
 $ git push origin mburke5678-patch-1 
```

Is origin: [mburke5678:mburke5678-patch-1](https://github.com/mburke5678/taxonomy/tree/mburke5678-patch-1)?

**Comment by jjasghar** [comment]
Can you show me the most recent command you used to push? 

When it comes to GitHub I'm limited in the amount of help I can give you due to the different ways one can set it up. Please understand that.

**Comment by mburke5678** [comment]
> Is origin: [mburke5678:mburke5678-patch-1](https://github.com/mburke5678/taxonomy/tree/mburke5678-patch-1)?
@jjasghar Yes.

**Comment by jjasghar** [comment]
if you run `ssh git@github.com` on your command line what comes back?

**Comment by kelbrown20** [comment]
There might be an issue with the SSH key here in the repo, I think this might help https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

**Comment by jjasghar** [comment]
Closing per combined PR here: https://github.com/instructlab/taxonomy/pull/1151

**Pull request closed without merging**